ads/adpicker.md

```diff
@@ -0,0 +1,38 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law o
+r agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AdPicker
+
+Please visit our website (https://adpicker.net) for more details
+
+## Example
+
+```html
+<amp-ad width="300" height="250"
+    type="adpicker"
+    data-ph="kbj2l87p3j">
+</amp-ad>
+```
+
+## Configuration
+
+Supported parameters:
+
+**Required**
+- width:        width of ad area
```

Comment:
Please see my comment before
>FYI: It is not always the case that AMP require both width and height. Please refer to supported amp-ad doc here.

Is there a reason that you require `width` and `height` to be set? Or is there a reason that you want to document them here? Otherwise we would generally not advice you to document these two attribute here because they are not specific to your adpicker ad network. 

I'll leave the final decision to you.